### PR TITLE
LPD-18663 ClaySamplePortlet is reusing id's 

### PR DESCRIPTION
**Pull request by pat270**
https://liferay.atlassian.net/browse/LPD-18663

### CODE REVIEW

**Comment by pat270** [comment]
ci:test:sf

**Comment by pat270** [comment]
ci:test:clay

File: modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/partials/cards.jsp [hunk 1]
```diff
@@ -502,7 +501,7 @@ ClaySampleUserCard claySampleUserCard = new ClaySampleUserCard();
 	</clay:col>
 
 	<clay:col
-		id="image-card-icon-block"
+		data-qa-id="image-card-block"
```
[the remark this answers is not in the document]
Comment (pat270) [reply modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/partials/cards.jsp]: img-card-icon-block

**Comment by pat270** [comment]
ci:test:sf

**Comment by pat270** [comment]
ci:test:relevant

**Comment by pat270** [comment]
ci:test:clay

**Review by ethib137 — APPROVED** [review]
LGTM. Feel free to forward once you've verified any failing tests.

**Comment by pat270** [comment]
ci:forward

**Pull request closed without merging**